src/clusto/services/http.py

```diff
@@ -52,6 +54,56 @@ def unclusto(obj, prefetch_attrs=None):
         return {'/%s/%s' % (obj.type, obj.name): result}
     return str(obj)
 
+def mass_unclusto(lst, prefetch_attrs):
+    '''
+    A more efficient form of unclusto for lists of objects to be unclusto-ed
+    '''
+    # We want to fetch all attrs for all objects in lst at once, so divide and conquer by fetching
+    # attrs for objects that can have attrs, and simply unclusto basic types
+    prefetchable, nonprefetchable = classify_prefetchable(lst)
+
+    # Get all entity_ids for prefetching and build a mapping back to the base object
+    entity_map = {}
+    for obj in prefetchable:
+        entity_map[obj.entity.entity_id] = obj
+    entity_ids_to_prefetch = entity_map.keys()
+
+    # Get all attributes, filter by the ones we're interested in
+    attr_map = defaultdict(list)
+    all_prefetched_attrs = Attribute.query().filter(Attribute.entity_id.in_(entity_ids_to_prefetch)).all()
+    filtered_prefetched_attrs = []
+    for prefetch_attr in prefetch_attrs:
+        filtered_prefetched_attrs.extend(Driver.attr_filter(all_prefetched_attrs, **prefetch_attr))
+
+    # Build a list of attributes for each entity by entity_id
+    for attr in filtered_prefetched_attrs:
+        attr_map[attr.entity.entity_id].append(attr)
+
+    all_results = [unclusto(x) for x in nonprefetchable]
+    for k, v in entity_map.items():
+        attrs = []
+        for attr in attr_map[k]:
+            attrs.append(unclusto(attr))
+        all_results.append({'/%s/%s' % (v.type, v.name): { 'attrs': attrs }})
+    return all_results
+
+    
+
+
+def classify_prefetchable(lst):
+    '''
+    Takes a list of clusto objects, and returns a list of objects with
+    prefetchable attrs and a list of objects without prefetchable attrs.
+    '''
+    prefetchable = []
+    nonprefetchable = []
+    for item in lst:
+        if issubclass(item.__class__, Driver):
```

Comment:
could you add some comment on this? I didn't know that we can only prefetch driver